docs/ports.md

```diff
@@ -128,46 +128,46 @@
     - Sources
       - [YARN default configuration](https://github.com/TOSIT-FR/hadoop/blob/5b1fa3e1aabb6bd350bf5dbd4f4e49578d94738f/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml)
 - NodeManager
-  - Container Manager   
+  - Container Manager
     The address of the container manager in the NodeManager. Access is typically granted to admins, and Dev/Support teams.
     - Port: 0 (default for dynamic port allocation) or 45454 (static port by convention)
     - Protocol: RPC
     - Property: `yarn.nodemanager.address`
     - External access: yes
     - Sources
       - [YARN default configuration](https://hadoop.apache.org/docs/r3.1.1/hadoop-yarn/hadoop-yarn-common/yarn-default.xml)
-  - Localizer   
+  - Localizer
     Address where the localizer IPC is. It is responsible for downloading and copying remote resources on the local filesystem.
     - Port: 8040
     - Protocol: RPC
     - Property: `yarn.nodemanager.localizer.address`
     - External access: no
     - Sources
       [YARN default configuration](https://hadoop.apache.org/docs/r3.1.1/hadoop-yarn/hadoop-yarn-common/yarn-default.xml)
-  - HTTPS server   
+  - HTTPS server
     WebUI server of the NodeManager for administrator and developers.
     - Port: 8044
     - Protocol: HTTPS
     - Property: `yarn.nodemanager.webapp.https.address`
     - External access: yes
     - Sources
       - [YARN default configuration](https://hadoop.apache.org/docs/r3.1.1/hadoop-yarn/hadoop-yarn-common/yarn-default.xml)
-  - ApplicationMaster   
-    Ephemeral HTTPS port are open by each ApplicationMaster. The tdp-collection default range is 40000-50000 but is parameterised in the `tdp_var_defaults/hadoop/hadoop.yml` `yarn.app.mapreduce.am.job.client.port-range` property.These ports are required to coordinates YARN tasks. They don't need to be accessible outside of the cluster but can can be if the permitted by the network firewall. The YARN ResourceManager route the requests. *Note that this configuration on holds true for mapreduce jobs, not spark*.
-    - Port: 40000-50000
+  - ApplicationMaster
+    Ephemeral HTTPS ports are opened by each ApplicationMaster. The `tdp-collection` default port range is unrestricted but is parameterized in the `tdp_var_defaults/hadoop/hadoop.yml` inventory file under the `yarn.app.mapreduce.am.job.client.port-range` property. Ports within this range can be accessible from outside the cluster if the permitted by the network firewall. *Note that this only restricts the port range used for mapreduce jobs, not spark*.
```

Comment:
Instead of ", not spark*.", use "*. In the case of Spark, refer to the Spark Driver port documentation, such as the `spark.driver.port` property".